web-client/src/app/components/header/header.component.scss

```diff
@@ -0,0 +1,8 @@
+#titleHeaderLabel {
+  display: block;
+}
+@media screen and (max-width: 260px) {
```

Comment:
Since we are using tailwind, we can [extend](https://tailwindcss.com/docs/screens#adding-smaller-breakpoints) the breakpoints to the desired size and apply the class to the element